samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs

```diff
@@ -128,13 +132,32 @@ public class SemanticKernelController : ControllerBase
         return this.Ok(new AskResult { Value = result.Result, Variables = result.Variables.Select(v => new KeyValuePair<string, string>(v.Key, v.Value)) });
     }
 
+    /// <summary>
+    /// Register authenticated OpenAPI skills with the planner's kernel.
+    /// If the request includes an auth header for an OpenAPI skill, register the skill with the kernel.
+    /// Else, don't register the skill as it'll fail on auth.
+    /// </summary>
+    private async Task RegisterOpenApiSkillsAsync(CopilotChatPlanner planner, OpenApiSkillsAuthHeaders openApiSkillsAuthHeaders)
```

Comment:
This is to register the Github skill, or the OpenAPI skill?

Or is it registering the Github skill as an OpenAPI skill?

Reply:
Github also uses the open api schema; We will be adding jira here as well; This is basically where you register skills with the kernel with a call like ImportOpenApiSkillFromFileAsync(...)
I think it will be less confusing when we also have jira in here.

Reply:
@glahaye we can hop on a call to clarify this more, but there's a couple of overloaded references here. 

- We have the actual OpenAPI SK Skill that takes an Open API / Swagger spec and parses through it to make a new skill and functions out of the API definitions. 
- These imported skills are referred as OpenAPI Skills, i.e., GitHub OpenAPI Skills. To help differentiate it from the skill defined in (1), we started referring to these as Plugins, so GitHub Plug-in, which is the frontend terminology.

In short, the OpenAPI SK Skill parses and imports from OpenAPI swagger to make Plug-in (OpenAPI Skill). 

You might also hear the plugins referenced as connectors but that got way too confusing with the actual connector skills, so we introduced the 'plug-in' concept

Reply:
Got it!